content/logs/java.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Java log Collection
```

Comment:
Is this Java or Log4J?

Reply:
Java, Log4j, Log4j2 and slf4j are logging lib